src/main/java/org/apache/accumulo/testing/continuous/UndefinedAnalyzer.java

```diff
@@ -174,21 +163,16 @@ public boolean contains(String row) {
       // The command may need to be adjusted if formatting changes.
 
       File dir = new File(acuLogDir);
-      File[] masterLogs = dir.listFiles(new FilenameFilter() {
-        @Override
-        public boolean accept(File dir, String name) {
-          return name.matches("load_events.log");
-        }
-      });
+      File[] masterLogs = dir.listFiles((dir1, name) -> name.matches("load_events.log"));
 
       SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss,SSS");
 
       if (masterLogs != null) {
         for (File masterLog : masterLogs) {
 
-          BufferedReader reader = new BufferedReader(new InputStreamReader(new FileInputStream(masterLog), UTF_8));
           String line;
-          try {
+          try (BufferedReader reader = new BufferedReader(new InputStreamReader(
```

Comment:
May be able to use `Files.newBufferedReader()` here.  It is a new method in Java 1.8